src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
To be perfectly clear: you PR is about "fixing realistic lighting" so I'd prefer you to take the shortest path to do this, instead of planning for the future.

I won't oppose you any longer on this point and if you insist I don't mind merging this code - especially since PR #239 will rework layers anyway.